script/slideshowTime.js

```diff
@@ -37,4 +39,15 @@ function fadeOut(slideshow) {
   slideshow.style.opacity = 0.5;
 }
 
+function createButton() {
+  let slideshow = document.getElementById("slideshow");
+  let button = document.createElement("button");
+  button.innerHTML = "Se Hybler";
+  button.addEventListener("click", hyblerLink);
+  slideshow.appendChild(button);
+}
+function hyblerLink() {
+  window.location = "hybler.html";
+}
```

Comment:
Det kan være mere digg å bruke et `a` element for linken istedet for å mekke javascript funksjon som bytter side. Da fungerer linken på en mer "vanlig" måte. Så du burde lage en 

```javascript
let buttonContainer = document.createElement("a");
buttonContainer.href = "/hybler.html"
```
og legge knappen inni der

Reply:
Yuppp, skal fikse